integration-cli/docker_utils.go

```diff
@@ -1564,6 +1564,7 @@ func setupRegistry(c *check.C) *testRegistryV2 {
 		if err = reg.Ping(); err == nil {
 			break
 		}
+		c.Log(err)
```

Comment:
I think the assert after the loop will do that.  And if so, @runcom why isn't that assert good enough?  Does the err change during the 50 tries?